s3curl.pl

```diff
@@ -119,24 +135,30 @@
   --copySrcRange {startIndex}-{endIndex}
   --createBucket [<region>]   create-bucket with optional location constraint
   --head                      HEAD request
+  --noexec                    show curl command but do not execute it
+  --querystringauth <days>    use query string auth, with Expires=time(now) + <days>
+                              You can also use <integer> with d,h,m,s suffix for an
+                              arbitrary interval. e.g. 72h for 72 hours.
+  --ordinarysigning           use ordinary signing case even when hostname is not known
   --debug                     enable debug logging
   --servicePath               service path which is not part of resource
   --endpoint                  add endpoint to be excluded from signed string. Specify multiple parameters if you need add more than one.
  common curl options:
   -H 'x-amz-acl: public-read' another way of using canned ACLs
   -v                          verbose logging
-USAGE
+";
+#USAGE
 die $usage if $help || !defined $keyId;
 
 if ($cmdLineSecretKey) {
-    printCmdlineSecretWarning();
-    sleep 5;
+#    printCmdlineSecretWarning();
+#    sleep 5;
 
     $secretKey = $cmdLineSecretKey;
 } else {
     my $keyinfo = $awsSecretAccessKeys{$keyId};
-    die "I don't know about key with friendly name $keyId. " .
-        "Do you need to set it up in $DOTFILE?"
+    die "I don't know about key with friendly name $keyId.\n" .
```

Comment:
this change seems unnecessary - let's keep changes to upstream at a minimum - or make a compelling argument for superficial changes like this